src/routes/landing/landing.less

```diff
@@ -700,6 +699,8 @@ tr.row.report {
   height: 85px;
   width: 100px;
   overflow: hidden;
+  position: absolute;
+  top: 50%;
```

Comment:
![image](https://github.com/petabencana/riskmap/assets/39596102/1e1bea87-ecdd-4d58-a897-97960d0cfa0d)

Still overlapping , try to give top as `85px` instead of in percentage